web-app/src/app/router/Router.tsx

```diff
@@ -17,8 +17,34 @@ import TermsAndConditions from '../screens/TermsAndConditions';
 import PrivacyPolicy from '../screens/PrivacyPolicy';
 import Feed from '../screens/Feed';
 import Feeds from '../screens/Feeds';
+import { SIGN_OUT_TARGET } from '../constants/Navigation';
+import {
+  LOGIN_CHANNEL,
+  LOGOUT_CHANNEL,
+  createDispatchChannel,
+} from '../services/channel-service';
+import { useAppDispatch } from '../hooks';
+import { logout } from '../store/profile-reducer';
 
 export const AppRouter: React.FC = () => {
+  const navigateTo = useNavigate();
+  const dispatch = useAppDispatch();
+  const logoutUser = (): void => {
+    dispatch(
+      logout({ redirectScreen: SIGN_OUT_TARGET, navigateTo, propagate: false }),
+    );
+  };
+
+  const loginUser = (): void => {
+    // Refresh the page to ensure the user is authenticated
```

Comment:
I would update `Refresh the page to ensure the user is authenticated on this session and other sessions (tabs)`

Reply:
The refresh will only affect the current _window/session_